packer/ansible/rackhd_ci_builds.yml

```diff
@@ -9,11 +9,10 @@
     - rabbitmq
```

Comment:
when you rename rackhd_package.yml it's not safe , but there're somewhere refer to this file. ( example packer/README.md , fit_tests/deploy/rackhd_source_install.py )